### PR TITLE
Added OTLP/HTTP JSON Exporter

### PR DESCRIPTION
**Pull request by riticksingh**
Signed-off-by:Ritick Gautam<riticksinghrajput@gmail.com>

### CODE REVIEW

**Comment by prondubuisi** [comment]
Hello @riticksingh did you get an email from EasyCLA confirming your signature signed? If so do confirm that you are using the same email for your Github push

**Comment by riticksingh** [comment]
> Hello @riticksingh did you get an email from EasyCLA confirming your signature signed? If so do confirm that you are using the same email for your Github push. This is what my email looked like
> EASYCLA

yes I got the mail from easyCLA on the same email id which I use for github but still it is showing missing ID commit

**Comment by bobstrecansky** [comment]
Looking at your individual commits, it looks like you've used a different email address in your local author's git config.  I checked out your branch locally, and I see the author on your last 3 commits being listed as `ritick@ritick.gautam` rather than `riticksinghrajput@gmail.com`.  I think this is the cause of the CLA check failing.  Output below:

```
$ git log -n 3

commit 740351c8a4aef48bac6eb27251f393198e998aab (HEAD -> ritickmaster, ritick/master)
Author: riticksingh <ritick@ritick.gautam>
Date:   Wed Nov 4 17:11:52 2020 +0530

    Updated OTLPExporterTest

commit e00896e491565f6148871e8aa160036c221dfcef
Author: riticksingh <ritick@ritick.gautam>
Date:   Wed Nov 4 16:45:27 2020 +0530

    Added test case for OTLPExporter

    Signed-off-by:Ritick Gautam

commit be556ad13fc4d9e3191c7cce7b5bdebd7bc7fd61
Author: riticksingh <ritick@ritick.gautam>
Date:   Wed Nov 4 14:40:45 2020 +0530

    Added OTLP/HTTP JSON Exporter

    Signed-off-by:Ritick Gautam<riticksinghrajput@gmail.com>
```

**Comment by bobstrecansky** [comment]
You can set your author name and email using the following tutorial:
https://www.git-tower.com/learn/git/faq/change-author-name-email/

File: contrib/otlp/Exporter.php [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    {
+
+        // Set default values based on presence of env variable
+        $this->endpointURL = getenv("OTEL_EXPORTER_OTLP_ENDPOINT") ?: "localhost:55680";
+        $this->protocol = getenv("OTEL_EXPORTER_OTLP_PROTOCOL") ?: "grpc";
+        $this->insecure = getenv("OTEL_EXPORTER_OTLP_INSECURE") ?: "false";
+        $this->certificateFile = getenv("OTEL_EXPORTER_OTLP_CERTIFICATE") ?: "none";
+        $this->headers[] = getenv("OTEL_EXPORTER_OTLP_HEADERS") ?: "none";
+        $this->compression = getenv("OTEL_EXPORTER_OTLP_COMPRESSION") ?: "none";
+        $this->timeout = getenv("OTEL_EXPORTER_OTLP_TIMEOUT") ?: 10;
```
Comment (dkarlovi) [inline contrib/otlp/Exporter.php:87]: `?:` is unsafe here since it will fail with `0`.

https://3v4l.org/lmWjD

Comment (bobstrecansky) [reply contrib/otlp/Exporter.php:87]: You're free to change this - it's your PR 👍

Comment (dkarlovi) [reply contrib/otlp/Exporter.php:87]: Eh, it might be a case of mistaken identity here. :)

Comment (bobstrecansky) [reply contrib/otlp/Exporter.php:87]: Sorry, was referring to @riticksingh - I could see how that would be confusing!

Comment (dkarlovi) [reply contrib/otlp/Exporter.php:87]: NP :+1:

File: contrib/otlp/Exporter.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\otlp;
```
Comment (adhocore) [inline contrib/otlp/Exporter.php:5]: is it typo or really lowercase `otlp` ?

Comment (riticksingh) [reply contrib/otlp/Exporter.php:5]: lowercase

Comment (bobstrecansky) [reply contrib/otlp/Exporter.php:5]: Other languages also use lower case, but I'm wondering if it's more PHP like to use capitals here.  I don't have strong opinions.

**Review by bobstrecansky — APPROVED** [review]
one comment on naming but otherwise this looks good to me!  Will let other contributors chime in.

File: contrib/otlp/Exporter.php (hunk 2 above)
Comment (adhocore) [reply contrib/otlp/Exporter.php:5]: imo StuldyCase is preferred more than camelCase or lowercase in class name and inamespace.

Comment (bobstrecansky) [reply contrib/otlp/Exporter.php:5]: StudlyCase would be fine with me too:

`namespace OpenTelemetry\Contrib\Otlp;`

Comment (riticksingh) [reply contrib/otlp/Exporter.php]: updated the namespace have a look and thanks for this :+1:

**Review by bobstrecansky — APPROVED** [review]

**Review by morrisonlevi — COMMENTED** [review]
Is the formatter running checks on this? The formatting for the constructor looked a bit weird, and there aren't trailing newlines in some of the files (which I thought was part of the checks, but maybe not).

**Comment by bobstrecansky** [comment]
Good catch @morrisonlevi - I ran php-cs-fixer on this using `make style` and noticed a couple fixes.

@riticksingh - can you perform that action and push it to this branch?  I'll open a separate issue to determine why this isn't being applied during CI.

**Pull request merged**